RiotSwiftUI/Modules/LocationSharing/StaticLocationSharingViewer/Test/Unit/StaticLocationViewingViewModelTests.swift

```diff
@@ -81,9 +81,9 @@ class StaticLocationViewingViewModelTests: XCTestCase {
     
     func testToggleShowUserLocation() {
         let viewModel = buildViewModel()
-        XCTAssertFalse(viewModel.context.viewState.showsUserLocation)
+        XCTAssertEqual(viewModel.context.viewState.showsUserLocationMode, .hide)
         viewModel.context.send(viewAction: .showUserLocation)
-        XCTAssertTrue(viewModel.context.viewState.showsUserLocation)
+        XCTAssertEqual(viewModel.context.viewState.showsUserLocationMode, .follow)
```

Comment:
What does it mean following a static location? 🤔

Reply:
the static location is the way the other user is sharing his location, but I want to follow my own movement in order to reach him